### DISCUSSION TITLE
How do you send email confirmation with react/react-native and Adonis?

### DISCUSSION THREAD

**Discussion by AndreCosta101** [adonisjs/core#1385] [Help] [answered]
I'm having a little trouble here. 

How would you go about receiving all data from the user, sending an email asking for confirmation and only then creating the entry in the database? Any ideas ? Thanks

**Comment by brandonsousa** [comment, marked as answer]
I would try to get all his data from the application, save it in the bank, have a column as active or confirmed starting in false, if it had been registered I would send an autologin link to his email, that is, with a token, when clicked it would update the active / confirmed column to true.

if he tried to login I would check if it was true or false, if true would return the login, if not the message "email needs to be confirmed"

**Reply by AndreCosta101** [reply]
Thanks so much Brandon!
